fasta/seeker.go

```diff
@@ -0,0 +1,100 @@
+package fasta
+
+import (
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"io"
+	"log"
+	"os"
+)
+
+// Seeker enables random access of fasta sequences using a pre-computed index.
+type Seeker struct {
+	file readSeekCloser
+	idx  Index
+}
+
+// Close the Seeker.
+func (rs *Seeker) Close() error {
+	return rs.file.Close()
+}
+
+// TODO
+// readSeekCloser is a TEMPORARY github actions is updated as the io.ReadSeekCloser
+// was added to the builtin io package in 2020.
```

Comment:
Am I correct in that this is extra code to make the code pass tests because the testing setup uses an older version of go?  Can we update the testing environment to use a newer version of Go?

Reply:
That is correct. I looked briefly into the problem, but could not figure out how to update the go version in the github action.